changelogs/fragments/51149-allow-merging-module-defaults-groups.yaml

```diff
@@ -0,0 +1,6 @@
+minor_changes:
+  - module_defaults - Deprecate module_defaults overwriting modules/groups defined in an outer scope by adding
+    a new MODULE_DEFAULTS_MERGE config option. When the option is set to True, module_defaults defined in an
+    inner scope are merged with those in the outer scope. module_defaults for a module/group may be emptied
+    by providing an empty dictionary. In 2.12 the default of MODULE_DEFAULTS_MERGE will be changed from
+    False to True.
```

Comment:
Update this to refer to the new `__clear__` key instead of an empty dictionary.

Reply:
Good catch. I cleaned up the changelog to use the `deprecated_features` section too.